src/auth.rs

```diff
@@ -54,31 +54,36 @@ pub fn parse_basic_auth(
 }
 
 /// Verify authentication
```

Comment:
Can you document the new tendency of this function to check out whether _any_ of the required passwords match? It might seem obvious but it should still be documented.

Reply:
@svenstaro I have changed it, check it out.